packages/create-strapi-app/utils/prompt-user.js

```diff
@@ -0,0 +1,114 @@
+'use strict';
+
+const inquirer = require('inquirer');
+const axios = require('axios');
+const yaml = require('js-yaml');
+
+/**
+ * @param {string|null} projectName - The name/path of project
+ * @param {string|null} template - The Github repo of the template
+ * @returns Object containting prompt answers
+ */
+module.exports = async function promptUser(projectName, template) {
+  const questions = await getPromptQuestions(projectName, template);
+  return inquirer.prompt(questions);
+};
+
+/**
+ *
+ * @returns Prompt question object
+ */
+async function getTemplateQuestion() {
+  const content = await getTemplateData(`templates/templates.yml`);
+
+  // Fallback to manual input when fetch fails
+  if (!content) {
+    return {
+      type: 'input',
+      message: 'Please provide the GitHub URL for your template:',
+    };
+  }
+
+  const choices = content.map(option => {
+    const name = option.title.replace('Template', '');
+    return {
+      name,
+      value: `https://github.com/${option.repo}`,
+    };
+  });
+
+  return {
+    type: 'list',
+    message: `Select a template`,
+    pageSize: choices.length,
+    choices,
+  };
+}
+
+/**
+ *
+ * @returns Array of prompt question objects
+ */
+async function getPromptQuestions(projectName, template) {
+  const templateQuestion = await getTemplateQuestion();
+
+  return [
+    {
+      type: 'list',
+      name: 'quick',
+      message: 'Choose your installation type',
+      choices: [
+        {
+          name: 'Quickstart (recommended)',
+          value: true,
+        },
+        {
+          name: 'Custom (manual settings)',
+          value: false,
+        },
+      ],
+    },
+    {
```

Comment:
the project name should always go if I remeber well

Reply:
updated, but it made sense to me to only confirm projectName when `--template` or `starter` was not also provided.  If those arguments are also found then just ask quickstart and continue.  wdyt?

Reply:
My message was unclear. I meant it should be the first question if no name was provided :D

Reply:
updated!